compiler/packages/snap/src/compiler.ts

```diff
@@ -128,6 +129,12 @@ function makePluginOptions(
   if (firstLine.includes("@disableMemoizationForDebugging")) {
     disableMemoizationForDebugging = true;
   }
+  if (firstLine.includes("@enableChangeDetectionForDebugging")) {
+    enableChangeDetectionForDebugging = {
+      source: "react-compiler-runtime",
```

Comment:
fyi you might already know this but we don't currently publish this package on npm, so this flag will only work internally for now

Reply:
Yeah, I wanted to include the runtime code as basically an example of/documentation for what the change detection code would look like. I've been updating it internally separately